pkg/config/profile.go

```diff
@@ -21,6 +21,7 @@ type Profile struct {
 	LiveModePublishableKey string
 	TestModeAPIKey         string
 	TestModePublishableKey string
+	TerminalPosDeviceID    string
```

Comment:
nit: should we do `TerminalPOSDeviceID` (capitalize POS)?

Reply:
yes it totally should be 👍